crates/matrix-sdk-base/src/room/room_info.rs

```diff
@@ -464,6 +464,13 @@ pub struct RoomInfo {
     /// more accurate than relying on the latest event.
     #[serde(default)]
     pub(crate) recency_stamp: Option<u64>,
+
+    /// A timestamp remembering when we observed the user accepting an invite on
+    /// this current device.
+    ///
+    /// This is useful to remember if the user accepted this a join on using
+    /// this specific client.
+    pub(crate) invite_accepted_at: Option<MilliSecondsSinceUnixEpoch>,
```

Comment:
I never do know how these work: does it need a serde default or some special migration to not break old instances?

Reply:
No: https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=dfb0cddf843b3f7eb2bda51fe094b5e1.

A missing field or a `null` are treated the same if `Option` is used. I guess we could add a `skip_serializing` here since this field will in most cases be `None`.

Reply:
> No: https://play.rust-lang.org/?version=stable&mode=debug&edition=2024&gist=dfb0cddf843b3f7eb2bda51fe094b5e1

Heh, nice, thank you! 👍